libs/ast_generic/AST_generic_helpers.mli

```diff
@@ -5,6 +5,7 @@ val str_of_ident : AST_generic.ident -> string
 exception NotAnExpr
 
 val expr_to_pattern : AST_generic.expr -> AST_generic.pattern
+val expr_to_stmt : AST_generic.expr -> AST_generic.stmt
```

Comment:
maybe a small comment that if the expr was an StmtExpr, we actually return the embedded stmt (and not generate an ExprStmt (StmtExpr ...)